### PR TITLE
Cleanup rules.mk for 32U4 keyboards, L-M

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

L and M.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/model01/keymaps/default/rules.mk [hunk 1]
```diff
@@ -1,2 +1 @@
-# used by default keymap
-MOUSEKEY_ENABLE = yes       # Mouse keys(+4700)
+MOUSEKEY_ENABLE = yes
```
Comment (noroadsleft) [inline keyboards/model01/keymaps/default/rules.mk]: Could you move this up to `keyboards/model01/rules.mk` please?

File: keyboards/miuni32/keymaps/default/rules.mk [hunk 2]
```diff
@@ -1,18 +1,3 @@
...
-MIDI_ENABLE = no            # MIDI controls
-AUDIO_ENABLE = no           # Audio output on port C6
-UNICODE_ENABLE = no         # Unicode
-BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
-RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight. 
-SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
-
+CONSOLE_ENABLE = no
+NKRO_ENABLE = yes
+RGBLIGHT_ENABLE = yes
```
Comment (noroadsleft) [inline keyboards/miuni32/keymaps/default/rules.mk]: Could you move these up to `keyboards/miuni32/rules.mk` please?

File: keyboards/lets_split/rev1/rules.mk [hunk 3]
```diff
@@ -1 +1 @@
-BACKLIGHT_ENABLE = no
\ No newline at end of file
+BACKLIGHT_ENABLE = no
```
Comment (noroadsleft) [inline keyboards/lets_split/rev1/rules.mk]: This file could be zero-length; Backlight is already disabled at keyboard level. Same for `keyboards/lets_split/rev2/rules.mk` also (which is unedited by this PR).

File: keyboards/launchpad/keymaps/default/rules.mk [hunk 4]
```diff
@@ -1,22 +1,2 @@
...
-AUDIO_ENABLE = no          # Audio output on port C6
-UNICODE_ENABLE = no         # Unicode
-BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
-RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight. 
-
-# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
-SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
-
+COMMAND_ENABLE = no
+NKRO_ENABLE = yes
```
Comment (noroadsleft) [inline keyboards/launchpad/keymaps/default/rules.mk]: Could these be moved to keyboard or revision level?

**Comment by fauxpark** [comment]
@noroadsleft done :)

**Pull request merged**